src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -67,6 +69,42 @@ public String description()
         }
     }
 
+    public enum ClientAuth
+    {
+        REQUIRED("true"),
+        NOT_REQUIRED("false"),
+        OPTIONAL("optional");
+        private final String value;
+        private static final Map<String, ClientAuth> VALUES = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
```

Comment:
giving this a second thought. It looks like the values are always lowercased, so I think we should go back to the original HashMap implementation

Reply:
Reverted it back